proto/leaderboard.proto

```diff
@@ -59,17 +55,20 @@ message _Element {
   // An id can only appear in a leaderboard one time. You can't have 2 scores for 1 player,
   // unless that player has 2 ids!
   uint32 id = 1;
-
   // The value by which this element is sorted within the leaderboard.
-  float score = 2;
+  double score = 3;
+
+  reserved 2;
```

Comment:
i think that is correct; if you want to avoid breaking backward compat with those SDKs (or related canaries) I think you would just want to keep the float field around and rename it to `deprecated32bitScore` or something. Doesn't matter all that much to me; @PDXKimani or @kvcache might have thoughts.

Reply:
This would be fine for clients.

For the service we will need to populate and support both, for a time. We can chat in more detail on slack if you like.